LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/JoinRequestMessageHandler.cs

```diff
@@ -62,6 +64,7 @@ async Task ProcessJoinRequestAsync(LoRaRequest request)
                     this.logger.LogInformation("join request received");
 
                     loRaDevice = await this.deviceRegistry.GetDeviceForJoinRequestAsync(devEUI, devNonce);
+                    this.deviceLoadRequests?.Add(1);
```

Comment:
Why not move this one inside the `LoRaDeviceRegistry` as well, then we have all `DeviceLoadRequests` metrics in one place?

Reply:
Done in 1ccb5eff69ce23adc0649c27b60ade46eface220